Items/04_00_WeaponAttachments.txt

```diff
@@ -474,45 +508,58 @@ This allows three point slings to be used without getting caught on your gear.
 Action. Allows you to carry the gun on your back. Holstering your weapon is a 
 Free Action.
 
-Strength Requirement:
-*  -1
-
-Range Bracket Miss Chance:
-*  -0.5
+Strength Requirement:  -1
+Range Bracket Miss Chance:  -0.5
 
 ===== Level 10 Requirement =====
 
 == Pistol Smart-Lanyard Holster System  $250 ==
+
     Dropping your weapon holsters it as free action. Drawing it takes 1 Action 
 as normal.
 
 ==============================
 Ammunition / Magazine Accessories
 ==============================
-    Only extended, magpull, and Drum mags apply to unconventional weapons 
-(weapons other than bullet-firing.) Total mag price should be calculated by 
-taking multipliers for extended mags first, then multipliers for special effect, 
-then magazine accessories like magpulls. Multipliers are multiplied together but 
-they do not multiply the constant.
-
-    For example, extended mag + poison bullet magazine magazine cost is 
-(1.3x * 2x + 50) = (2.6x+50)
 
 ===== Level 1 Requirement =====
 
 == Extended Magazines  $(1.3x) Mag cost. ==
+
      $12 mags now cost $15, etc. Round Up. Adds 30% magazine capacity to 
 magazine/clip-based weapons. Revolver systems or rocket launchers, for example, 
 are excluded.
 
 == Hollow Point Magazine  $1.5x + 10 mag cost ==
-    +30% base damage, -1 Armor Piercing Level
+
+    Hollow point ammunition are bullets that have either a hollow tip or a tip
+with water inside. This makes the bullet fragment more dramatically on impact,
+which means that it penetrates armor less effectively. However, if it hits 
+something squishy like fruit, a person's body or the delecate silicone and boron

```

Comment:
sp: "delicate"

Reply:
I swear in front of my Lord God Most High that I did at some point attend an English language class in public school!